torchtext/vocab.py

```diff
@@ -427,6 +427,26 @@ def cache(self, name, cache, url=None, max_vectors=None):
             logger.info('Loading vectors from {}'.format(path_pt))
             self.itos, self.stoi, self.vectors, self.dim = torch.load(path_pt)
 
+    def __len__(self):
+        return len(self.vectors)
+
+    def get_vecs_by_tokens(self, tokens, lower_case_backup=False):
+        to_reduce = False
```

Comment:
It would be great if you could add some docs for function arguments. A simple example could be there as well. See an example here. https://github.com/pytorch/text/blob/3b7b0e2b6bb0a60d955dfb5647c084a26f1d8e44/torchtext/data/dataset.py#L326-L341

Reply:
I have added docs and test. But the check failed.
```
CondaHTTPError: HTTP 504 GATEWAY_TIMEOUT for url <https://conda.anaconda.org/pytorch/noarch/current_repodata.json>
Elapsed: 00:59.414826
CF-RAY: 4fa17e2018b3c510-ORD
A remote server error occurred when trying to retrieve this URL.
A 500-type error (e.g. 500, 501, 502, 503, etc.) indicates the server failed to
fulfill a valid request.  The problem may be spurious, and will resolve itself if you
try your request again.  If the problem persists, consider notifying the maintainer
of the remote server.
```